boxwise_flask/graph_ql/resolvers.py

```diff
@@ -30,38 +32,40 @@ def serialize_date(value):
 # registers this fn as a resolver for the "allBases" field, can use it as the
 # resolver for more than one thing by just adding more decorators
 @query.field("allBases")
-def resolve_all_camps(_, info):
+def resolve_all_bases(_, info):
     # discard the first input because it belongs to a root type (Query, Mutation,
     # Subscription). Otherwise it would be a value returned by a parent resolver.
-    response = Bases.get_all_bases()
+    response = Base.get_all_bases()
     return list(response.dicts())
 
 
 # not everyone can see all the bases
 # see the comment in https://github.com/boxwise/boxwise-flask/pull/19
 @query.field("orgBases")
 def resolve_org_bases(_, info, org_id):
-    response = Bases.get_camps_by_org_id(org_id)
+    response = Base.get_for_organisation(org_id)
```

Comment:
I find this less descriptive than it was previously, but matter of preference

Reply:
actually, I guess this comment belongs in the Base model, but you get what I mean